src/lambda/commands/deploySamApplication.ts

```diff
@@ -26,19 +33,34 @@ export async function deploySamApplication(
 
     const tempFolder = await mkdtemp('samDeploy')
     const outputTemplatePath = path.join(tempFolder, 'template.yaml')
-
+    let stage = 'packaging'
     try {
         const packageInvocation = new SamCliPackageInvocation(template.fsPath, outputTemplatePath, s3Bucket, invoker)
+        restParams.outputChannel.show(true)
+        // TODO: Add nls support
+        restParams.outputChannel.appendLine(`Packaging SAM app to "${s3Bucket}" S3 bucket`)
         await packageInvocation.execute()
 
+        stage = 'deploying'
         const deployInvocation = new SamCliDeployInvocation(outputTemplatePath, stackName, invoker)
+        // Deploying can take a very long time for Python Lambda's with native dependencies so user needs feedback
+        restParams.outputChannel.appendLine(localize(
+          'AWS.samcli.deploy.stackName.initiated',
+          'Deploying "{0}" stack...',
+          stackName
+        ))
         await deployInvocation.execute()
-
+        // TODO: Add nls support
+        const msg = `Successfully deployed "${stackName}" to S3 "${s3Bucket}"`
```

Comment:
Minor: Consider: 
`Successfully deployed SAM Application to CloudFormation Stack: ${stackName}`

At that point, I don't know if bucket is relevant. It is simply a mechanism to host the sam build artifact for actual deployment.

Reply:
✔